recipes/miniz/all/conanfile.py

```diff
@@ -46,6 +46,8 @@ def _configure_cmake(self):
         return self._cmake
 
     def build(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
         cmake = self._configure_cmake()
         cmake.build(target=self.name)
```

Comment:
```suggestion
        cmake.build()
```
It's a residue I left (in the first commit, files were installed manually instead of using `cmake.install()` in order to avoid building everything).

Reply:
thanks for the hint!